code/datums/mutations.dm

```diff
@@ -646,6 +646,60 @@ GLOBAL_LIST_EMPTY(mutations_list)
 	if(owner.a_intent == INTENT_HARM)
 		owner.LaserEyes(target)
 
+/datum/mutation/human/healing_hands
+	name = "Healing Hands"
+	quality = POSITIVE
+	get_chance = 27.5
+	lowest_value = 256 * 12
+	text_gain_indication = "<span class='notice'>You feel a comforting buzzing in your hands.</span>"
+	time_coeff = 5
+	var/next_heal_time = 0
```

Comment:
Mutation datums are singletons, so everyone who has the mutation uses the same datum instance. If you do it this way, everyone who has the mutation will share the same cooldown. If you want everyone to have their own cooldown, you will have to store it in a var on the mob or in an associative list on the datum.